bin/tex2stree

```diff
@@ -49,8 +53,9 @@ mjAPI.typeset({
 }, function (data) {
   if (!data.errors) {
     if (argv.xml) {
-      console.log(data.streeXml);}
+      console.log(argv.min ? data.streeXml : data.pprintXml(data.streeXml));}
```

Comment:
It seems a bit odd to me to pass the function back in the data object.  Why not pass the `min` parameter in the typeset call and have `GetSemantic` send back the form that the user wants?  (I would recommend making the parameter to `mj-single` a bit more descriptive, like `minSTree` or something, since `min` is a bit generic (though the command-line argument of `--min` is fine).